packages/itwinui-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -560,7 +578,11 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
                 }
               />
             </>
-            <ComboBoxEndIcon disabled={inputProps?.disabled} isOpen={isOpen} />
+            <ComboBoxEndIcon
+              disabled={inputProps?.disabled}
+              isOpen={isOpen}
+              {...endIconProps}
```

Comment:
Should `endIconProps` be before `isOpen`?

Reply:
Updated!